app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -491,7 +491,7 @@ def create_relationship_tree
 
       host_folder = {
         :name          => 'host',
-        :is_datacenter => false,
+        :type          => EmsFolder.name,
```

Comment:
Yeah I'd seen the `class.name`  in a lot of inv refresh code but wasn't sure the benefit.  I'll switch to just the string.